src/assets/scripts/client/aircraft/StripView/StripViewModel.js

```diff
@@ -506,7 +492,7 @@ export default class StripViewModel extends BaseModel {
     _redraw() {
         this.$callsignView.text(this._callsign);
         this.$aircraftTypeView.text(this._aircraftType);
-        this.$cidView.text(this.cidString);
+        this.$cidView.text(this._cid);
```

Comment:
should be cidString, not cid

Reply:
Is now natively a string, so _cid is good again.